src/types/domain.d.ts

```diff
@@ -685,6 +704,14 @@ export interface Assessment {
   updated: string;
 }
 
+export interface PagedAssessments {
+  content: Assessment[];
+  totalPages: number;
+  totalElements: number;
+  size: number;
+  number: number;
+}
```

Comment:
Generics to the rescue:

```suggestion
export interface Page<T> {
  content: T[];
  totalPages: number;
  totalElements: number;
  size: number;
  number: number;
}
```

Reply:
7ae5bb7